src/algorithms/algorithm-v1.ts

```diff
@@ -1,23 +1,25 @@
+import {DrawAlgorithm} from "./interface";
+
 class TryAgainAlgorithm implements DrawAlgorithm {
 
     private numbers: number[];
-    private drawedNumbers: number[];
+    private drawnNumbers: number[];
 
     constructor(numbers:number[]) {
         this.numbers = numbers;
-        this.drawedNumbers = new Array();
+        this.drawnNumbers = [];
     }
 
     drawNumber(): number {
         // Draws a number from the original array
-        // Keep trying until it wasn't already drawed
-        var number: number;
+        // Keep trying until it wasn't already drawn
+        let number: number;
```

Comment:
let and const are confusing for me in TS. In Swift, let is a const and var is, well, a variable.